src/main/java/com/ericsson/ei/handlers/MongoDBHandler.java

```diff
@@ -342,4 +343,46 @@ public void dropDatabase(String databaseName) {
         MongoDatabase db = mongoClient.getDatabase(databaseName);
         db.drop();
     }
+    
+    /**
+     * Check if the document exists
+     * @param databaseName
+     * @param collectionName
+     * @param condition is a condition for update documents
+     * @return
+     */
+    public boolean checkObjectExists(String databaseName, String collectionName, String condition) {
+    	MongoDatabase db = mongoClient.getDatabase(databaseName);
+        MongoCollection<Document> mongoCollection = db.getCollection(collectionName);
+        Document doc = mongoCollection.find(BasicDBObject.parse(condition)).first();
+        if (doc == null || doc.isEmpty()) {
+        	return false;
+        }
+    	return true;
+    }
+    
+    
+    /**
+     * Update the existing doucment with unique objects list
+     * @param dataBaseName
+     * @param collectionName
+     * @param condition is a condition for update documents
+     * @param eventId eventId to update in the mapper collection
+     * @return 
+     */
+    public boolean updateDocumentAddToSet(String dataBaseName, String collectionName, String condition, String eventId) {
+        try {
+            MongoCollection<Document> collection = getMongoCollection(dataBaseName, collectionName);
+            if (collection != null) {
+                final Document dbObjectInput = Document.parse(condition);
+                UpdateResult updateMany = collection.updateOne(dbObjectInput, Updates.addToSet("objects", eventId));
```

Comment:
This method is specific to the event to object map collection, right? Perhaps clarify this in the javadoc so that it's not used elsewhere.

Reply:
done